src/Illuminate/Database/composer.json

```diff
@@ -19,7 +19,8 @@
         "ext-json": "*",
         "illuminate/container": "^7.0",
         "illuminate/contracts": "^7.0",
-        "illuminate/support": "^7.0"
+        "illuminate/support": "^7.0",
+        "symfony/console": "^4.0 || ^5.0"
```

Comment:
Only ^5.0 will do. 4.0 support is dropped for Laravel 7

Reply:
Please rebase this PR against the current master branch, so you are working with the Symfony 5 version of the codebase.